apps/nova/src/components/Settings/ProfileForm.tsx

```diff
@@ -0,0 +1,367 @@
+  const topics = form.watch("topics");
+
+  const canSubmit = useMemo(() => {
+    const currentValues = { city, name, email, password, phoneNumber, topics };
+    const initialValues = form.control._defaultValues;
+
+    if (photo) return true;
+    if (!isEqual(currentValues, initialValues)) return true;
+    return false;
+  }, [
+    photo,
+    form.control._defaultValues,
+    city,
+    name,
+    email,
+    password,
+    phoneNumber,
+    topics,
+  ]);
+
+  const onSuccess = useCallback(() => {
+    invalidateQuery([QueryKey.FindCurrentUser]);
+  }, [invalidateQuery]);
+
+  const onError = useCallback(
+    (error: Error) => {
+      toast.error({
+        title: intl("profile.update.error"),
+        description: error.message,
+      });
+    },
+    [intl, toast]
+  );
+
+  const cityArr = useMemo(
+    () =>
+      Object.entries(governorates).map(([key, value]) => ({
+        label: intl(value),
+        value: IUser.City[key as keyof typeof IUser.City],
+      })),
+    [intl]
+  );
+
+  const profileMutation = useUpdateUser({ onSuccess, onError });
+
+  const notifyComingSoon = useCallback(() => {
+    toast.success({
+      title: intl("settings.notifications.coming-soon.title"),
+      description: intl("settings.notifications.coming-soon.description"),
+    });
+  }, [intl, toast]);
+
+  const onSubmit = useCallback(
+    (data: IForm) => {
+      if (!canSubmit) return;
+      if (photo) console.log("upload photo");
+      if (!user) return;
+
+      const addTopics: number[] = topics.filter(
+        (topic) => !userTopics.includes(topic)
+      );
+      const removeTopics: number[] = userTopics.filter(
+        (topic) => !topics.includes(topic)
+      );
+
+      profileMutation.mutate({
+        id: user.id,
+        payload: {
+          name: data.name && data.name !== user.name ? data.name : undefined,
+          email:
+            data.email && data.email !== user.email ? data.email : undefined,
+          phoneNumber: data.phoneNumber ? data.phoneNumber : undefined,
+          password:
+            data.password.new && data.password.current
+              ? {
+                  new: data.password.new,
+                  current: data.password.current,
+                }
+              : undefined,
+          city: data.city && data.city !== user.city ? data.city : undefined,
+          addTopics,
+          removeTopics,
+        },
+      });
+    },
+    [profileMutation, photo, user, canSubmit, topics, userTopics]
+  );
+
+  useEffect(() => {
+    if (!requirePassword) {
+      form.trigger("password.current");
+      form.trigger("password.new");
+      form.trigger("password.confirm");
+    }
+  }, [form, requirePassword]);
+
+  return (
+    <Form
+      onSubmit={form.handleSubmit(onSubmit)}
+      className="grid justify-items-stretch grid-cols-2 gap-x-28 gap-y-8 p-14"
+    >
+      <div className=" flex flex-col gap-6">
+        <UploadPhoto setPhoto={setPhoto} photo={photo || orNull(user?.image)} />
```

Comment:
![image](https://github.com/user-attachments/assets/ab6b42c9-445b-4a50-83d0-1f8f9c875934)

This part doesn't match the design.